.github/workflows/lint-test-build-push.yaml

```
@@ -0,0 +1,207 @@
+name: Lint, Test, Build, Push
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+
+jobs:
+  lint-test-build-push:
+    runs-on: ubuntu-latest
+    env:
+      # Determine whether or not we should build the
+      # final production image and push it to GHCR.
+      production_build: ${{ github.event_name != 'pull_request' &&
+        github.ref == 'refs/heads/sebastiaan/backend/cache-docker-images' }}
+
+    steps:
+      # Create a short SHA-tag to tag built images
+      - name: Create SHA Container Tag
+        id: sha_tag
+        run: |
+          tag=$(cut -c 1-7 <<< $GITHUB_SHA)
+          echo "::set-output name=tag::$tag"
+      - name: Checkout code
```

It's more readable with a space between the steps.
```suggestion

      - name: Checkout code
```

---

This was included, but not recognized as such by GH, in one of the commits I pushed.